src/Internal/QueryM/Ogmios.purs

```diff
@@ -1153,6 +1163,51 @@ type ChainPoint =
   , hash :: OgmiosBlockHeaderHash
   }
 
+---------------- ADDITIONAL UTXO MAP REQUEST
+
+newtype AdditionalUtxoSet = AdditionalUtxoSet OgmiosUtxoMap
+
+derive newtype instance Show AdditionalUtxoSet
+
+type OgmiosUtxoMap = Map OgmiosTxOutRef OgmiosTxOut
+
+instance EncodeAeson AdditionalUtxoSet where
+  encodeAeson' (AdditionalUtxoSet m) =
+    encodeAeson' $ encode <$> utxos
+
+    where
+    utxos :: Array (OgmiosTxOutRef /\ OgmiosTxOut)
+    utxos = Map.toUnfoldable m
+
+    encode (inp /\ out) =
+      { "txId": inp.txId
+      , "index": inp.index
+      }
+        /\
+          { "address": out.address
+          , "datumHash": out.datumHash
+          , "datum": out.datum
+          , "script": out.script
+          , "value":
+              { "coins": out.value # valueToCoin # getLovelace
+              , "assets": out.value # getNonAdaAsset # encodeNonAdaAsset
+              }
+          }
+
+    encodeNonAdaAsset assets = encodeMap $
```

Comment:
Missing a type signature (also applies to some bindings below here). Even when the type is obvious, our convention is to always include the explicit signature